projects/valtimo/config/assets/core/nl.json

```diff
@@ -47,7 +47,7 @@
       "saveSuccess": "Uw aantekening is succesvol opgeslagen."
     },
     "zaakobjecten": {
-      "objecttype": "Objecttype",
+      "objecttype": "Selecteer het type Object dat je wilt zien",
```

Comment:
Selecteer het type object dat je wilt zien ('object' without capital except for German where all nouns are capitalized; the Objects API also does not capitalize the word object: https://objects-and-objecttypes-api.readthedocs.io/en/latest/)